recipes/fs-access-mode-constants/src/workflow.ts

```diff
@@ -0,0 +1,284 @@
+import type { Edit, SgRoot } from "@codemod.com/jssg-types/main";
+import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
+import { getNodeImportStatements } from "@nodejs/codemod-utils/ast-grep/import-statement";
+import type Js from "@codemod.com/jssg-types/langs/javascript";
+
+export default function tranform(root: SgRoot<Js>): string | null {
```

Comment:
Is there perhaps a type for this function you can apply via annotation, like `const transform: Workflow = (root) => {`?